main.go

```diff
@@ -72,8 +73,22 @@ const (
 	AccountEmpty
 )
 
+// Result is the action taken by a swipe of a card. It contains all information
+// to be communicated to the user.
+type Result struct {
+	Code    ResultCode
+	UID     []byte
+	User    string
+	Account float32
```

Comment:
Why is this a float32? Anything will cost 1€, so this could be an int. If this ever changes, we have to touch the code anyways.